cmd/forwarder/run/run.go

```diff
@@ -154,8 +155,19 @@ func (c *command) runE(cmd *cobra.Command, _ []string) (cmdErr error) {
 	if c.mitmConfig.CACertFile != "" {
 		c.mitm = true
 	}
+	if len(c.mitmDomains) > 0 {
+		c.mitm = true
+	}
```

Comment:
How about we 'or' it with the condition above?

Reply:
Fixed that.